loops.js

```diff
@@ -24,7 +26,15 @@
 // loop...
 // klaar!
 // ==========================================
+for(let i=1 ; i <= 5 ;i++){
+    if(i === 5){
+        console.log("klaar");
+    } else {
+        let a = "loop";
```

Comment:
De if-else structuur om "loop" te printen voor de eerste vier iteraties en "klaar" bij de laatste is duidelijk en efficiënt. De code is goed leesbaar en functioneel.